kcbq-connector/src/main/java/com/wepay/kafka/connect/bigquery/config/BigQuerySinkConfig.java

```diff
@@ -346,7 +414,14 @@ private String sanitizeTableName(String tableName) {
    *         name if sanitization is enabled.
    */
   public String getTableFromTopic(String topic) {
-    return getBoolean(SANITIZE_TOPICS_CONFIG) ? sanitizeTableName(topic) : topic;
+    String tableName = null;
+    Map<String, String> map = getTopicsToTables();
```

Comment:
OK, I'm resigned to this. Two options:

1. A util class (e.g. TopicToTableResolver) with a static method that takes a config and returns the map.
2. Cache the map in the config object

I'm partial to (1). The SinkConnector would use the util method to get the map for ensureExistingTables, and the SinkTask would use the util method to get the map and store it as an instance variable to be used in the class.

How's that sound?